packs/orion/actions/lib/actions.py

```diff
@@ -176,8 +223,10 @@ def get_ncm_transfer_results(self, transfer_id, sleep_delay=10):
         """
         ts = {}
         while True:
```

Comment:
Would also be a good idea to add a timeout to this function directly? Or you think action (runner) level timeout is sufficient?